tests/google/adwords.py

```diff
@@ -68,3 +68,67 @@ def test_adwords_data_upload_for_offline_conversion(self):
         print(uploaded, failed)
         self.assertEqual(len(uploaded), 2)
         self.assertEqual(len(failed), 1)
+
+    def test__should__be_able_to_get_all_campaigns__with_one_account(self):
```

Comment:
do you use tags to mark integration tests which can be skipped?

Reply:
no, I name the test file without a test prefix accordingly which skips it in unit tests, but is able to run from within the IDE